books/RayTracingInOneWeekend.html

```diff
@@ -1429,8 +1430,56 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [rtweekend-initial]: <kbd>[rtweekend.h]</kbd> The rtweekend.h common header]
 
+All main program files will include `rtweekend.h` first, so most other header files (where the bulk
+of our code will reside) can assume that these definitions are already available. (Headers included
+inside `rtweekend.h` still need to include any of their dependencies.) We'll make some updates with
```

Comment:
Starting a sentence with a parenthetical feels a bit odd.  I think we can just omit the parenthesis all together an have it be a standalone sentence.